README.md

```diff
@@ -1,7 +1,7 @@
 [![][kong-logo]][kong-url]
 
-[![Build Status][badge-travis-image]][badge-travis-url]
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Kong/kong/blob/master/LICENSE)
+[![Build Status](https://img.shields.io/travis/Kong/kong?logo=Travis%20ci)](https://travis-ci.org/Kong/kong/branches)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?logo=Github)](https://github.com/Kong/kong/blob/master/LICENSE)
```

Comment:
Why does `Apache License` have `Github logo?

Reply:
Oh sorry!! 
I can fix it

Reply:
Fixed it!! @bungle